app/services/preserved_object_handler.rb

```diff
@@ -76,7 +76,7 @@ def check_existence
           incoming_vers_eq_pc_vers = (incoming_version == pres_copy.version)
           incoming_vers_gt_pc_vers = (incoming_version > pres_copy.version)
           found_expected_version = (incoming_vers_eq_pc_vers || incoming_vers_gt_pc_vers)
-          check_status_if_not_ok(pres_copy, found_expected_version)
+          check_status_if_not_ok_or_unexpected_version(pres_copy, found_expected_version)
```

Comment:
I'm wondering if the code is clearer if you take each comparator relationship separately below and call `check_status_if_not_ok_or_unexpected_version(pres_copy, false)` or true as the case may be.  

The reason I suggest this is the comment with line 100 about ran_moab_validation setting -- it feels like there is some slight of hand going on now ...

But have yet to look at the whole -- still going commit by commit ...